types/lhr.d.ts

```diff
@@ -21,10 +21,21 @@ declare global {
       credits: Record<string, string>,
     }
 
+    /**
+     * The full output of a Lighthouse flow. Includes a series of Lighthouse runs.
+     * TODO(FR-COMPAT): Add flow specific metadata (e.g. Overall flow score).
```

Comment:
```suggestion
     * TODO(FR-COMPAT): Add flow specific metadata (e.g. interaction steps).
```

that one is much closer than an overall flow score ;)